### DISCUSSION TITLE
Ahead and Behind Car Model on Track

### DISCUSSION THREAD

**Discussion by Weini1972** [kaiusl/KLPlugins.DynLeaderboards#28] [Ideas]
Thank you very much for your great work. How about displaying the car model of at least 5 cars in front and 5 cars behind on track of me in a leaderboard? Would be great. Thanks again

**Comment by kaiusl** [comment]
Do you mean on the provided dashboard? If so then the answer is unfortunately no. The dashboard provided with this plugin is meant for mobile screens and there is not enough space for much more data on it. Furthermore it is also just meant to be an example on how to use this plugin and it is encouraged for everyone to customize it to their needs and wishes or create completely new ones.

Thus you'll need to edit and customize the dashboard yourself. For car model names you'll need to enable the `Car.Model` property in the settings and edit the dashboard. If you need any help with it, let me know, I'm happy to try and help.

**Reply by Weini1972** [reply]
Thank you for your feedback. My idea was to display Cars Behind and Ahead On Track in the leaderboard as a variant to your current leaderboard. If I understand correctly, this is currently not possible by switching on the car model in the settings. It would be great if this could be realized. Let me know if I can help you with the implementation.
Thank you very much

**Reply by kaiusl** [reply]
I'm now not sure I understand completely but cars behind and ahead on track are shown by the RelativeOnTrack leaderboard type, to which you can simply switch to using `DynLeaderboardsPlugin.Dynamic.NextLeaderboard` (or PreviousLeaderboard) SimHub actions.

**Reply by Weini1972** [reply]
The idea for implementation should look like this
<img width="595" alt="Screenshot 2024-01-30 201523" src="https://github.com/kaiusl/KLPlugins.DynLeaderboards/assets/96842558/25247163-618b-4312-9ee7-caf2c9889d93">
The yellow line is me as driver and fix on this leaderboard position. The drivers in front and behind are dynamic and refer to the positions of the cars on the track, not to their position in the race. Hope it is better explained now? Is it realizable like this?

**Reply by kaiusl** [reply]
I think RelativeOnTrack leaderboard type does this, no?

**Reply by Weini1972** [reply]
So far I haven't managed to do that. I'll try again in a minute. Many thanks so far

**Reply by kaiusl** [reply]
Okay, let me know how it goes. :)
